Cargo.toml

```diff
@@ -11,16 +11,27 @@ documentation = "https://docs.rs/raft"
 description = "The rust language implementation of Raft algorithm."
 categories = ["algorithms", "database-implementations"]
 edition = "2018"
+build = "build.rs"
+
+[build-dependencies]
+protoc-rust = "~2.0-2.2"
+protoc-grpcio = "0.3.1"
```

Comment:
It is very strange to depend on a gRPC lib

Reply:
It shouldn't be necessary, using protobuf-build should be enough